### ISSUE TITLE
Add extra HF shower library for 2011-2012 reprocessing

### ISSUE THREAD

**Issue by civanch** [cms-sw/cmsdist#795]
A new shower HF library file needs to be added to HCAL data:

 /afs/cern.ch/user/k/kheyn/public/For_Salavat/HFShowerLibrary_oldpmt_eta4_16en.root

This SL will be customized for 2011-2012 reprocessing.

Other two existing files with SL should not be removed.

**Comment by davidlange6** [comment]
as just discussed in the ORP - please clarify why we can not remove the "old" run1 shower library (used 5 years ago) - it of course will still be available in all old releases…

**Comment by civanch** [comment]
As discussed at Sim meeting during CMS week new SL for 2011-2012 reprocessing should be rebuilt, so this PR should be on hold.

**Comment by civanch** [comment]
HF experts have prepared a new version of Sl for 2011-2012 reprocessing:

/afs/cern.ch/user/a/abdullin/public/HF_NewShoweLibrary_OldPMTs/HFShowerLibrary_oldpmt_noatt_eta4_16en.root

please, include this file to HCAL data.

**Comment by ktf** [comment]
Ciao, @degano can you take care of this? @civanch can you specify which package is this for?

**Comment by civanch** [comment]
This HF SL is used from SImG4CMS/Calo/data at SIM step. This particular SL is prepared for legacy simulation for 2011-2012 runs within 7_1_X and 7_2_X. The default SL for legacy reproduction should not be removed, so this file is 3d variant of HF SL (post LS1, old legacy, new legacy).

**Comment by davidlange6** [comment]
Hi Vladimir

But since we have not run production since ~2008(??) with the "old" HF SL, what legacy reproduction are we keeping it for?

**Comment by civanch** [comment]
Hi David,

Yes, your are right. However, we discussed before summer the same question and I proposed to substitute old by new, there was no consensus. At he same time, there was no strong argument to keep the old one.... We may drop the old file at any time.

**Comment by davidlange6** [comment]
Since it is an case available in many releases I request we drop it. It's already a lot to need to carry around two shower libraries

On Sep 1, 2014, at 3:58 PM, "Vladimir Ivantchenko" <notifications@github.com<mailto:notifications@github.com>> wrote:

Hi David,

Yes, your are right. However, we discussed before summer the same question and I proposed to substitute old by new, there was no consensus. At he same time, there was no strong argument to keep the old one....
